index.js

```diff
@@ -168,6 +171,22 @@ class Conf {
 		this.events.emit('change');
 	}
 
+	_ensureDirectory() {
+		// Ensure the directory exists as it could have been deleted in the meantime
+		makeDir.sync(path.dirname(this.path));
+	}
+
+	_watch() {
```

Comment:
I'm leaning towards making this a public method, that way we can return the watcher/a disposable that way the watcher can get cleaned up. Curious what your opinion on this is.

Reply:
I would prefer to wait until someone actually requests that.